pkg/audit/audit_formatter.go

```diff
@@ -28,47 +31,77 @@ import (
 // AuditFormatter is the interface definition for the formatter used by the auditLogServer
 type AuditFormatter interface {
 	NewHumanReadableEvent(context.Context, *esApi.Event) *audit.HumanReadableEvent
+	NewUserOverview(context.Context, *projections.User) *audit.UserOverview
 }
 
 // auditFormatter is the implementation of AuditFormatter used by auditLogServer
 type auditFormatter struct {
 	log        logger.Logger
 	esClient   esApi.EventStoreClient
 	efRegistry ef.EventFormatterRegistry
+	// TODO: replace with qhDomain -> sooner or later we will probably need all repos
+	// 	especially when building overviews
+	// 	or use no repos and build snapshots from the store
```

Comment:
making a dependency on qhDomain doesn't sound all that right either, maybe list all needed repos for now and try to abstract later in the future? (after for example all overviews are implemented)

Reply:
@jastBytes reminded me about an important detail, where we don’t want to get all users overview rather all users overview **at a specific timestamp** and therefore repos are not needed anymore. Snapshots will be created according to the timestamp